course_discovery/apps/course_metadata/models.py

```diff
@@ -1275,6 +1276,10 @@ class CourseRun(DraftModelMixin, CachedMixin, TimeStampedModel):
 
     invite_only = models.BooleanField(default=False)
     featured = models.BooleanField(default=False)
+    marketing_price = models.BooleanField(default=False, verbose_name=_('Price'), help_text=_(
+        'Indicates whether the course on marketing site is paid'))
+    marketing_price_value = models.CharField(max_length=255, null=True, blank=True, verbose_name=_('Price Value'))
+    marketing_hide_price = models.BooleanField(default=False, verbose_name=_('Hide Price'))
```

Comment:
this could be `is_marketing_price_visible`. The names should be intuitive and simple

Reply:
The field name are same which are set in the WordPress